Jenkinsfile

```diff
@@ -41,21 +46,26 @@ node ("ts-module && heavy && java8") {
         }
     }
 
+    stage('Integration Tests') {
+        sh './gradlew integrationTest'
+        junit testResults: 'build/test-results/integrationTest/*.xml', allowEmptyResults: true, healthScaleFactor: 0.0
+    }
+
     stage('Analytics') {
-        sh './gradlew check spotbugsmain javadoc'
+        sh './gradlew check -x test spotbugsmain'
```

Comment:
```suggestion
        sh './gradlew check --exclude-task test spotbugsmain'
```

Reply:
Would it be a thought to just drop the `check` task entirely and just run `gradlew spotbugsmain pmd checkstyle` or whichever the right task names are? Those could potentially be put into a parallel block later as well as individual steps.

If we eventually add new analytics we'd just have to add another block with the new task. Not sure if `check` really contains anything we wouldn't want to call out explicitly. Less mystery that way.

Will try to test this PR shortly.

Reply:
hm, isn't there this weird split about `spotBugsMain` and `spotBugTests` for everything, and there is no general `spotbugs` task to just run both? :thinking:

Reply:
Erg, yeah that may be. Although maybe that's just two tasks then. In any case that's a trivial thing we can worry about later if we want to :-)

Works fine!